aggregator/types.go

```diff
@@ -25,16 +25,23 @@ import (
 	"time"
 
 	"github.com/m3db/m3aggregator/aggregation/quantile/cm"
+	"github.com/m3db/m3metrics/metric/id"
 	"github.com/m3db/m3metrics/policy"
 	"github.com/m3db/m3metrics/protocol/msgpack"
 	"github.com/m3db/m3x/clock"
 	"github.com/m3db/m3x/instrument"
 	"github.com/m3db/m3x/pool"
 )
 
-// ShardFn maps a id to a shard given the total number of shards.
+// ShardFn maps a id to an aggregator shard given the total number of shards.
 type ShardFn func(id []byte, numShards int) uint32
 
+// PartitionFn maps a chunked id to a backend partition.
+type PartitionFn func(chunkedID id.ChunkedID) uint32
```

Comment:
I know we use partition for these kinds of use cases, but can we unify them to "shard" in m3db repos?

Reply:
Can take a look to unify the naming convention, I intentionally called it something else to distinguish agg tier shards and backend shards, but that's fair too.